src/Microsoft.Azure.SignalR.Common/Auth/AuthUtility.cs

```diff
@@ -65,8 +65,8 @@ public static string GenerateAccessToken(byte[] keyBytes,
         return jwtToken.Length > MaxTokenLength ? throw new AzureSignalRAccessTokenTooLongException() : jwtToken;
     }
 
-    public static string GenerateRequestId()
+    public static string GenerateRequestId(string traceIdentifier)
     {
-        return Convert.ToBase64String(BitConverter.GetBytes(Stopwatch.GetTimestamp()));
+        return $"{traceIdentifier}-{Convert.ToBase64String(BitConverter.GetBytes(Stopwatch.GetTimestamp()))}";
```

Comment:
traceIdentifier is in such format `0HN7T92QVLV5T:00000009` https://github.com/aspnet/KestrelHttpServer/blob/master/src/Kestrel.Core/Internal/Infrastructure/CorrelationIdGenerator.cs#L19, and could be customized using middlewares. 
Do we want `:` in the requestId? might the generated requestId too long?

Reply:
now I use `return $"{traceIdentifier}:{Stopwatch.GetTimestamp().ToString("X")}";`